packages/desktop-client/src/components/common/View.tsx

```diff
@@ -16,13 +16,13 @@ const View = (props: ViewProps) => {
   // everywhere and we can avoid any perf penalty that glamor would
   // incur.
 
-  const { style, nativeStyle, innerRef, ...restProps } = props;
+  const { style, nativeStyle, innerRef, className = '', ...restProps } = props;
```

Comment:
THANK YOU. This must have been responsible for all those `undefined` classnames I was seeing.

Reply:
yup :D